tests/integration/targets/vmware_ops_manage_folder_test/tasks/main.yml

```diff
@@ -1,64 +1,65 @@
 ---
-- name: Import common vars for tests on vCenter Environment
-  ansible.builtin.include_vars:
-    file: ../group_vars.yml
-  when: not run_on_simulator
+- name: Run Manage folder tests
+  block:
+    - name: Import common vars for tests on vCenter Environment
+      ansible.builtin.include_vars:
+        file: ../group_vars.yml
+      when: not run_on_simulator
 
-- name: Create Folder Tree
-  loop: "{{ folder_types }}"
-  ansible.builtin.include_role:
-    name: cloud.vmware_ops.manage_folder
-  vars:
-    manage_folder_state: present
-    manage_folder_folder_name: production/foo/web
-    manage_folder_folder_type: "{{ item }}"
+    - name: Create Folder Tree
+      loop: "{{ folder_types }}"
+      ansible.builtin.include_role:
+        name: cloud.vmware_ops.manage_folder
+      vars:
+        manage_folder_state: present
+        manage_folder_folder_name: production/foo/web
+        manage_folder_folder_type: "{{ item }}"
 
-- name: Create Folder Without Managing Full Tree
-  loop: "{{ folder_types }}"
-  ansible.builtin.include_role:
-    name: cloud.vmware_ops.manage_folder
-  vars:
-    manage_folder_state: present
-    manage_folder_folder_name: db
-    manage_folder_parent_folder: production/foo
-    manage_folder_folder_type: "{{ item }}"
+    - name: Create Folder Without Managing Full Tree
+      loop: "{{ folder_types }}"
+      ansible.builtin.include_role:
+        name: cloud.vmware_ops.manage_folder
+      vars:
+        manage_folder_state: present
+        manage_folder_folder_name: db
+        manage_folder_parent_folder: production/foo
+        manage_folder_folder_type: "{{ item }}"
 
-- name: Create A Folder With A Slash In It
-  loop: "{{ folder_types }}"
-  ansible.builtin.include_role:
-    name: cloud.vmware_ops.manage_folder
-  vars:
-    manage_folder_state: present
-    manage_folder_folder_name: security/syseng
-    manage_folder_parent_folder: production/foo
-    manage_folder_parse_name_as_path: false
-    manage_folder_folder_type: "{{ item }}"
+    - name: Create A Folder With A Slash In It
+      loop: "{{ folder_types }}"
+      ansible.builtin.include_role:
+        name: cloud.vmware_ops.manage_folder
+      vars:
+        manage_folder_state: present
+        manage_folder_folder_name: security/syseng
+        manage_folder_parent_folder: production/foo
```

Comment:
please add `tiny_prefix`, maybe better to create a new variable in vars/main.yml for it.
add `tiny_prefix` also to cleanup step [here](https://github.com/redhat-cop/cloud.vmware_ops/blob/main/tests/integration/targets/vmware_ops_manage_folder_test/tasks/main.yml#L68)

Reply:
done